doc/modules/language-guide/pages/motoko-introduction.adoc

```diff
@@ -24,14 +24,14 @@ Consequently, entire classes of common programming pitfalls that are common in o
 To execute, {proglang} statically compiles to link:about-this-guide{outfilesuffix}#wasm[WebAssembly], a portable binary format that abstracts cleanly over modern computer hardware, and thus permits its execution broadly on the Internet, and the link:../developers-guide/concepts/what-is-ic{outfilesuffix}#ic-overview[{IC}].
 
 [[pitch-actors]]
-== Each canister smart constract as an _actor_
+== Each canister smart contract as an _actor_
 
 {proglang} provides an *actor-based* programming model to developers to express _services_, including those of canister smart contracts on the link:../developers-guide/concepts/what-is-ic{outfilesuffix}#ic-overview[{IC}].
 
 An actor is similar to an object, but is special in that its state is completely isolated, and all its interactions with the world are by _asynchronous_ messaging.
 
 All communication with and between actors involves passing messages asynchronously over the network using the Internet Computer's messaging protocol.
-An actor’s messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating `await`s).
+An actor’s messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating `await` s).
```

Comment:
Make `actor` teletype here? Also, are backticks allowed in `.adoc`?

Reply:
Im a bit confused about the usage of teletype, italics, bold and so on. 
I read the language guides ( https://smartcontracts.org/docs/language-guide/about-this-guide.html#_documentation_conventions ) but it seems the usage bold, italics, and so on, don't always comply with the guidelines and are being used in a way the author interpreted it the best.

Reply:
Single backticks seem to be _emphasis_.

Reply:
```suggestion
An actor's messages are processed in sequence, so state modifications never admit race conditions (unless explicitly allowed by punctuating ``await``s).
```